uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-request-record-actions.scss

```diff
@@ -0,0 +1,33 @@
+leave-request-record-actions {
+  display: block;
+
+  .fa,
+  .btn-group {
+    margin-right: $crm-gap-medium;
+  }
+}
+
+leave-request-record-actions .btn-toolbar {
```

Comment:
As I note in another comment, we shouldn't have a toolbar since its use would be incorrect, but aside from that i'm not sure why you really need it at all. You mentioned that it was there to avoid extra gaps, but this is how it looks like if i get rid of the toolbar and i remove that custom margin you added above
<img width="600" alt="no-toolbar" src="https://user-images.githubusercontent.com/6400898/37704955-b51c2ed0-2cfa-11e8-99d8-555fb0c15827.png">

Reply:
so couldn't you simplify the whole thing and just add a margin-right to the btn-group?

Reply:
This is how it looks to me without 'toolbar' and margin:

![image](https://user-images.githubusercontent.com/3973243/37707054-8d940d3e-2cf9-11e8-8997-2347d147ea39.png)

Reply:
Solved via floating.